### DISCUSSION TITLE
Chapter 2, exercise 46c

### DISCUSSION THREAD

**Discussion by zarry0** [spamegg1/Epp-Discrete-Math-5th-solutions#2] [Q&A] [answered]
In Exercise 46c from Chapter 2, you concluded that the two expressions were not logically equivalent according to the truth table. But I believe the truth values for the compound statement [(p ^ r) xor (q ^ r)] are wrong.

<img width="857" alt="Captura de pantalla 2023-05-08 a la(s) 22 57 17" src="https://user-images.githubusercontent.com/87690107/236998269-4f075ce5-e229-4d43-8a71-6b1fec65d094.png">

I'm sorry if this is not the right place to discuss this; I didn't know where else to reach you.

Anyway, thanks for all the hard work you've put into these solutions. They have been of great help when studying:)

**Comment by spamegg1** [comment]
It is the right place. Thanks for the feedback!

It's very possible that I made a mistake in the truth table, or have a typo in the conclusion. I'll double-check.

**Reply by spamegg1** [reply, marked as answer]
You are correct, they are equivalent, my truth table was wrong :smile:

**Reply by spamegg1** [reply]
Should be fixed now! Thanks again! :+1:
